react/features/toolbox/components/ToolbarButton.web.js

```diff
@@ -109,17 +106,17 @@ class ToolbarButton extends AbstractToolbarButton {
      */
     render(): ReactElement<*> {
         const { button } = this.props;
-        const attributes = getButtonAttributesByProps(button);
         const popups = button.popups || [];
 
+        const props = {
+            ...this.props,
+            createRefToButton: this._createRefToButton
+        };
+
         return (
-            <a
-                { ...attributes }
-                onClick = { this._onClick }
-                ref = { this._createRefToButton }>
-                { this._renderInnerElementsIfRequired() }
+            <StatelessToolbarButton { ...props }>
                 { this._renderPopups(popups) }
-            </a>
+            </StatelessToolbarButton>
```

Comment:
I named it "stateless" because it doesn't use this.state, lifecycle methods and redux.